### PR TITLE
Add detection for minor and major releases

### PR DESCRIPTION
**Pull request by lemieux**
This will allow us to deploy major and minor version through circle ci by creating release notes for that new version. This will only work for a clean major/minor version (to X.0.0 or X.X.0).

@jugarrit @dannytranlx @alavers @mspensieri

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -189,6 +189,25 @@ module.exports = function(grunt) {
         var versionType = grunt.option('versionType');
         var globalVersion;
 
+        // unless the version or increment is explicitely set, let's try
```
Comment (jugarrit) [inline Gruntfile.js]: nit: explicitly

File: Gruntfile.js [hunk 2]
```diff
@@ -189,6 +189,25 @@ module.exports = function(grunt) {
...
+        // to figure out what is the next version
+        if (!fullVersion && !versionType) {
+            var currentVersion = require('./package.json').version;
```
Comment (jugarrit) [inline Gruntfile.js]: const?

**Comment by jugarrit** [comment]
Sure 
👍

File: Gruntfile.js (hunk 2 above)
Comment (lemieux) [inline Gruntfile.js]: true I guess we all run node 4 at least

**Comment by jugarrit** [comment]
Sure 👍

**Comment by Mario54** [comment]
👍 🎉

**Pull request merged**